core_lib/src/tool/movetool.cpp

```diff
@@ -56,6 +58,31 @@ QCursor MoveTool::cursor()
     return mScribbleArea->currentTool()->selectMoveCursor(mode, type());
 }
 
+void MoveTool::updateSettings(const SETTING setting)
+{
+    switch (setting)
+    {
+    case SETTING::ROTATION_INCREMENT:
+    {
+        mRotationIncrement = mEditor->preference()->getInt(SETTING::ROTATION_INCREMENT);
+        break;
+    }
+    default:
+        break;
+
+    }
+}
+
+int MoveTool::rotationAngleIncrement()
+{
+    if (mCachedRotIncrement != mRotationIncrement)
+    {
+        mRotationIncrement = mEditor->preference()->getInt(SETTING::ROTATION_INCREMENT);
+        mCachedRotIncrement = mRotationIncrement;
+    }
+    return mCachedRotIncrement;
```

Comment:
Why do we need `mCachedRotIncrement`? Can't we just set `mRotationIncrement` after the connect and use that whenever we need the rotation angle increment?

Reply:
👍